src/terminal/parser/InputStateMachineEngine.cpp

```diff
@@ -386,9 +387,11 @@ bool InputStateMachineEngine::ActionCsiDispatch(const VTID id, const VTParameter
         DWORD buttonState = 0;
         DWORD eventFlags = 0;
         const size_t firstParameter = parameters.at(0).value_or(0);
+        const til::point uiPos{ parameters.at(1) - 1, parameters.at(2) - 1 };

```

Comment:
_mild worry_: if not enough params are passed, then `parameters.at(1) - 1` will get resolved as `-1`

Reply:
The ISME isn't great about parameter validation today. Alas!